spec/visual/samples/php

```diff
@@ -27,6 +27,89 @@ echo $ΔΔ;
 
 "thing {$thing->other_thing()}"
 
+function &byref() {
+    $x = array();
+    return $x;
+}
+
+trait SomeTrait {
+  // Some visibility
+  const PUBLIC_CONST_A = 1;
+  public const PUBLIC_CONST_B = 2;
+  protected const PROTECTED_CONST = 3;
+  private const PRIVATE_CONST = 4;
+      
+  function afunc(string $arg, SomeInterface $arg2, callable $arg3, object $arg4): void {
+    echo "hello";
+    if (1 <=> 1) {
+      echo "yep!";
+    }
+  }
+}
+
+interface SomeInterface {
+  function interfaceFunc(bool $arg): iterable;
+  function nullableTypes(?bool $arg): ?iterable;
+}
+
+//Anonymous class example
+$app->setLogger(new class implements Logger {
+    public function log(string $msg) {
+        echo $msg;
+    }
+});
+
+// some uses
+use some\namespace\ClassA;
+use some\namespace\ClassB;
+use some\namespace\ClassC as C;
+
+use function some\namespace\fn_a;
+use function some\namespace\fn_b;
+use function some\namespace\fn_c;
+
+use const some\namespace\ConstA;
+use const some\namespace\ConstB;
+use const some\namespace\ConstC;
+
+// Grouped uses
+use some\namespace\{ClassA, ClassB, ClassC as C};
```

Comment:
The second slash here is being rendered as an error token. Presumably that's not what you intended, right?

Reply:
This is one of the cases where I haven't finished the job, it needs more work to render this correctly, but as this is valid php code I've added the example.

Reply:
Just fixed this up.